docs/model-archiver.md

```diff
@@ -37,7 +37,7 @@ pip install .
 
 Now let's cover the details on using the CLI tool: `model-archiver`.
 
-Here is an example usage with the squeezenet_v1.1 model archive following the example in the [examples README](../examples/README.md):
+Here is an example usage with the squeezenet_v1.1 model archive following the example in the [examples README](https://github.com/pytorch/serve/tree/master/examples):
```

Comment:
@jlin27 Can we keep the links relative and not hardcode the path?

Reply:
Hi Geeta - This link will point to the examples section in the repo, rather than a set of documentation so that is why it is hardcoded. The reason it was a relative link in the original was because it was still referencing within the repo.